src/capture/parse_movie_es.cc

```diff
@@ -0,0 +1,69 @@
+#include <iostream>
```

Comment:
We should reuse parse_movie.cc, and change its behavior with a command line flag.